packages/paste-core/components/form/src/shared/Utils.tsx

```diff
@@ -1,9 +1,9 @@
-const PROPS_TO_BLOCK = ['className', 'style', 'size', 'height', 'width'];
+export const PROPS_TO_BLOCK: string[] = ['className', 'style', 'size', 'height', 'width'];
 
-export const safelySpreadFormControlProps = (props: {}): {} => {
+export const safelySpreadFormControlProps = (props: {}, propsToBlock: string[] = PROPS_TO_BLOCK): {} => {
```

Comment:
Going to end up suggesting what I suggested for the Button component this week. The props list is so small, lets just unset set them on the select and save the look up. e.g.

```
<SelectElement   
  {...props}
  className={undefined}
  style={undefined}       
  height={undefined}
  width={undefined}
  aria-invalid={hasError}
  ref={ref}
  multiple={multiple}
  size={multiple ? size : 0}
>

Reply:
Ok great, that makes sense. Will make this change.

Reply:
@SiTaggart Just checked out `<Button />`, any benefit to adding [similar PropType validation](https://github.com/twilio-labs/paste/blob/master/packages/paste-core/components/button/src/index.tsx#L7)? Or is this too much oversight for this component?

Reply:
We probably don't need to worry about doing the PropType validation here.

Reply:
@richbachman not even for `this.props.children`?

Reply:
You could add `children: NonNullable<React.ReactNode>;` to the `SelectProps`. Check out [Radio](https://github.com/twilio-labs/paste/blob/master/packages/paste-core/components/form/src/Radio.tsx#L15) for reference.

Reply:
Just added that, thank you!